lib/ddtrace/ext/http.rb

```diff
@@ -6,7 +6,9 @@ module HTTP
       METHOD = 'http.method'.freeze
       STATUS_CODE = 'http.status_code'.freeze
       TEMPLATE = 'template'.freeze
-      TYPE = 'http'.freeze
+      TYPE_INBOUND = 'web'.freeze
```

Comment:
nice, I like this distinction "inbound" vs "outbound", makes more sense than "web" vs "http"